src/macros.js

```diff
@@ -655,13 +655,10 @@ defineMacro("\\mod", "\\allowbreak" +
     "{\\rm mod}\\,\\,#1");
 
 // \pmb    --   A simulation of bold.
-// It works by typesetting three copies of the argument with small offsets.
-// Ref: a rather lengthy macro in ambsy.sty
-defineMacro("\\pmb", "\\html@mathml{\\@binrel{#1}{" +
-    "\\mathrlap{#1}" +
-    "\\mathrlap{\\mkern0.4mu\\raisebox{0.4mu}{$#1$}}" +
-    "{\\mkern0.8mu#1}" +
-    "}}{\\mathbf{#1}}");
+// The version in ambsy.sty works by typesetting three copies of the argument
+// with small offsets. We use two copies. We omit the vertical offset because
+// of rendering problems that makeVList encounters in Safari.
+defineMacro("\\pmb", "\\@binrel{#1}{\\mathrlap{#1}\\kern0.5px#1}");
```

Comment:
What about MathML output?

Reply:
This code produces MathML that also renders as a simulated bold.  In the screenshot below, the three versions are from MathJax, KaTeX, and KaTeX-MathML:
![pmb](https://user-images.githubusercontent.com/16403058/56304102-393a8780-60f2-11e9-86a2-9dfd76032aa6.PNG)

Reply:
@ronkok Currently MathML is used for accessibility, and I think it'd be better to render only once, maybe with `\mathbf` like before.

Reply:
Yes, MathML is currently used only for accessibility, but that will only be true until #1928 is resolved. More importantly, `\mathbf` is currently ineffective in KaTeX MathML. You can verify this by looking at the entry for `\pmb` in http://webmathcomparison.net/#p. 

I haven't yet investigated why this occurs, but I suspect some sort of bug similar to what I address in PR #1929, which applies classes such as `\mathbin` in MathML.

Reply:
@ylemkimon I have revised the MathML as you requested, using `\mathbf`. So this PR can be merged. But I consider the larger matter unfinished at least until `\mathbf` works in MathML.